packages/video_player/README.md

```diff
@@ -29,7 +29,7 @@ This package is not an _endorsed_ implementation of `video_player`. Therefore, y
 ```yaml
 dependencies:
   video_player: ^2.2.6
-  video_player_tizen: ^2.3.0
+  video_player_tizen: ^2.4.0
```

Comment:
The version should be 2.3.1 according to semantic versioning as this patch doesn't add new functionality.

Reply:
I'm increasing minor version because #283 is not published.

Reply:
https://github.com/flutter-tizen/plugins/pull/283 doesn't still add any functionality.

Reply:
I thought #283 as an improvement of code. (https://semver.org/#spec-item-7)
I'll fix it!